README.md

```diff
@@ -106,7 +106,17 @@ jobs:
           WAKATIME_API_KEY: ${{ secrets.WAKATIME_API_KEY }}
           GH_TOKEN: ${{ secrets.GH_TOKEN }}
           USERNAME: <username>
-          HEAD_FLAG: true
+          SHOW_TITLE: true
```

Comment:
Will it be good if:
```yml
SHOW_TITLE: true
```
was set as `false` in the template above Like this?

```yml
name: Waka Readme
on:
  schedule:
    # Runs at 12am UTC
    - cron: '0 0 * * *'
jobs:
  update-readme:
    name: Update this repo's README
    runs-on: ubuntu-latest
    steps:
      - uses: athul/waka-readme@master
        with:
          WAKATIME_API_KEY: ${{ secrets.WAKATIME_API_KEY }}
          SHOW_TITLE: false # default | optional - refer the Extras section
```

So they just have to toggle `true` / `false`

Reply:
Since its optional flasg its okay to omit it, but this example is a little more explanatory.